variables.tf

```diff
@@ -15,6 +15,14 @@ variable "auth_client_secret" {
   description = "Client Secret for GitHub Auth"
 }
 
+variable "log_level" {
+  description = "The level of log severity (e.g. DEBUG, INFO, ERROR etc)"
```

Comment:
This might be a good candidate for a default variable value that can be overridden